src/lesson3/task1/Loop.kt

```diff
@@ -114,7 +137,12 @@ fun squareBetweenExists(m: Int, n: Int): Boolean = TODO()
  * sin(x) = x - x^3 / 3! + x^5 / 5! - x^7 / 7! + ...
  * Нужную точность считать достигнутой, если очередной член ряда меньше eps по модулю
  */
-fun sin(x: Double, eps: Double): Double = TODO()
+fun sin(x: Double, eps: Double)
```

Comment:
Looks awfully :(. Use loop here please. Also, please try to check the case when `x = 20 * PI` or even `x = 100 * PI`.